service.go

```diff
@@ -269,13 +303,17 @@ func NewFrameBuffer() (*FrameBuffer, error) {
 
 	// Xvfb will print the display on which it is listening to file descriptor 3,
 	// for which we provide a pipe.
-	xvfb := exec.Command("Xvfb", "-displayfd", "3", "-nolisten", "tcp")
+	arguments := []string{"-displayfd", "3", "-nolisten", "tcp"}
+	if len(options.ScreenSize) > 0 {
```

Comment:
optional: Add validation that this string is of the form we expect.

I think a regular expression would be fine, something like:

```
if options.ScreenSize == "" {
   if !screenSizeRE.MatchString(`\d+x\d+(?:x\d+)`, options.ScreenSize) {
      return fmt.Errorf("invalid screen size: expected 'WxH[xD]', got %q", options.ScreenSize)
   }
}
```

I suppose we could split and parse the string to ensure the digits aren't 0, but that's a lot of lines of code for a small gain.